### DISCUSSION TITLE
How to show articles in card view as shown in Samples and Docs

### DISCUSSION THREAD

**Discussion by sardarms** [nunocoracao/blowfish#1086] [Q&A] [answered]
Sorry for the very basic question, but I am unable to have card view all over the sites like https://blowfish.page/samples/ and https://blowfish.page/docs/ 

![image](https://github.com/nunocoracao/blowfish/assets/83923880/3a91eb45-1c49-4513-9d8e-81c6fdc6a6b4)

But I am not sure what to change. 

I have tried to enable cardView on the params. Then on _index.md. But still the articles are listed as line with a cropped image. 

Can anyone please help?

**Comment by sardarms** [comment]
I figured it out. I used the below code in the _index.md and it solved the issue. 

```
cardView: true
showCards: true
```

**Comment by nunocoracao** [comment, marked as answer]
Hello @sardarms this will work but there is a more streamlined way if you want to enable it for everything. 
In your `params.toml` file there are separate sections for different areas of the site. All of them have their own cardView value. 

Check this page - https://blowfish.page/docs/configuration/#theme-parameters
Specifically:
Lists - https://blowfish.page/docs/configuration/#list
Taxonomy - https://blowfish.page/docs/configuration/#taxonomy
Term - https://blowfish.page/docs/configuration/#term
Homepage - https://blowfish.page/docs/configuration/#homepage

This allows users to chose how to render content in different ways across the site.
